tests/providers/aws/__init__.py

```diff
@@ -0,0 +1,20 @@
+SOME_AWS_REGIONS = (
+    'us-east-2',
+    'us-east-1',
+    'us-west-1',
+    'us-west-2',
+    'ap-northeast-1',
+    'ap-northeast-2',
+    'ap-northeast-3',
+    'ap-south-1',
+    'ap-southeast-1',
+    'ap-southeast-2',
+    'ca-central-1',
+    'cn-north-1',
+    'cn-northwest-1',
+    'eu-central-1',
+    'eu-west-1',
+    'eu-west-2',
+    'eu-west-3',
+    'sa-east-1'
+)
```

Comment:
Probably not necessary for testing purposes, but as an FYI, there is an API call to retrieve an authoritative list of regions - `ec2.describeRegions()`